_pytest/debugging.py

```diff
@@ -95,13 +95,17 @@ def _enter_pdb(node, excinfo, rep):
 
 
 def _postmortem_traceback(excinfo):
-    # A doctest.UnexpectedException is not useful for post_mortem.
-    # Use the underlying exception instead:
-    from doctest import UnexpectedException
-    if isinstance(excinfo.value, UnexpectedException):
-        return excinfo.value.exc_info[2]
-    else:
-        return excinfo._excinfo[2]
+    try:
+        from doctest import UnexpectedException
```

Comment:
I think we can move this to the top level now. At the time, as @RonnyPfannschmidt commented in https://github.com/pytest-dev/pytest/issues/1810#issuecomment-248931216, we could not import logging at the top level, but this is no longer the case since we integrated the logging plugin.



> it seems that doctest imports logging, thus breaking the expectation of the test